README.md

```diff
@@ -1,21 +1,68 @@
-# Website
+# Get Started
 
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
-## Installation
+Docusaurus is a **static-site generator**. It builds a **single-page application** with fast client-side navigation, leveraging the full power of **React** to make your site interactive. It provides out-of-the-box **documentation features** but can be used to create **any kind of site** (personal website, product, blog, marketing landing pages, etc).
 
-```console
-yarn install
+
+
+## Dependencies
+
+* **Nodejs** version >= 14 or above
+
+* **Yarn** version >= 1.5
+* **NPM** [Guide](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+
+
+## How to install
+
+Windows and Mac users download [Nodejs installer](https://nodejs.org/en/download/) from their Official website.																			
```

Comment:
so mac has package managers. It may be good to break it up with each package manager( macports and homebrew) as well as a section without a package manager